### DISCUSSION TITLE
Welcome to reasoning-engine Discussions!

### DISCUSSION THREAD

**Discussion by conormckenzie** [conormckenzie/reasoning-engine#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by AceJester90** [comment]
Hi! I'm Ace, I'm a college student currently on my graduating year. I'm currently working on improving my skills and would love any guidance or suggestions you may have. I mainly contribute by helping on reviewing PRs and doing whatever task is assigned to me. I look forward to contributing and learning alongside everyone 😁
